src/doc/trpl/hello-cargo.md

```diff
@@ -73,7 +74,8 @@ extra goodies. According to the TOML docs,
 
 [toml]: https://github.com/toml-lang/toml
 
-Once you have this file in place, we should be ready to build! To do so, run:
+Once you have this file in place in your project's root directory, we should be
```

Comment:
Same thing with the original text being bad, but this uses 'you' and 'we' both. They should both be 'we'